src/clientSideScene/ClientSideSceneComp.tsx

```diff
@@ -696,6 +696,15 @@ export const CamDebugSettings = () => {
           }
         }}
       />
+      <div>
+        <button
+          onClick={() => {
+            sceneInfra.camControls.resetCameraPosition()
+          }}
+        >
+          Reset Camera Position
+        </button>
+      </div>
```

Comment:
To clarify, I've moved the "Reset Camera Position" button from the gizmo handle to the debug panel.

Reply:
Nice I'll make a ContextMenu component that will override the default browser one where we need it, and I'll make the Gizmo have one with each of the options and "Reset camera position" as the first option.